extras/nightly-container/provision.yml

```diff
@@ -111,3 +113,35 @@
       args:
         warn: no
 
+    - name: Configure gluster-exporter to use glusterd2
+      replace:
+        path: /etc/gluster-exporter/gluster-exporter.toml
+        regexp: 'gluster-mgmt = "glusterd"'
```

Comment:
May want to make these matches a bit more robust ala: `'^gluster-mgmt =.*'` (or something alone those lines)... Just to ensure the replacement happens even if the default options changes subtly.

Reply:
Fixed as suggested.